maintainer/conda/MDAnalysis/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+MDA_USE_OPENMP=FALSE pip install package/
```

Comment:
Ok, we should document and mention this. It's actually a good reason for installing from source. 

--
Oliver Beckstein
email: orbeckst@gmail.com


Am Mar 17, 2016 um 1:19 schrieb kain88-de <notifications@github.com>:

> In maintainer/conda/MDAnalysis/build.sh:
> 
> > @@ -0,0 +1,4 @@
> > +#!/bin/bash
> > +
> > +MDA_USE_OPENMP=FALSE pip install package/
> Conda doesn't ship openmp libraries. To avoid that a release package is build with globally installed openmp versions we default to only serial builds on conda do ensure that the binary package truly runs anywhere.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>